dist/temml.d.ts

```diff
@@ -0,0 +1,42 @@
+export interface Options {
+  displayMode?: boolean;
+  annotate?: boolean;
+  leqno?: boolean;
+  throwOnError?: boolean;
+  errorColor?: string;
+  macros?: Record<string, string>;
+  wrap?: "tex" | "=";
```

Comment:
Please revise to `wrap?: "tex" | "=" | "none";` 

`none` is a valid value. I'll update the documentation in the next release.